.travis.yml

```diff
@@ -41,36 +41,39 @@ jobs:
     - script: SCALA_VERSION=2.12.10 bin/test-2.1x
       name: "Run tests for Scala 2.12"
     - script: SCALA_VERSION=2.12.10 bin/test-2.1x
-      env: TRAVIS_JDK=11
-      name: "Run tests for Scala 2.12 and Java 11"
+      env: TRAVIS_JDK=8
+      name: "Run tests for Scala 2.12 and Java 8"
     - script: SCALA_VERSION=2.13.0 bin/test-2.1x
       name: "Run tests for Scala 2.13"
     - script: SCALA_VERSION=2.12.10 bin/test-multi-jvm-2.1x
       name: "Run multi-jvm tests for Scala 2.12"
     - script: SCALA_VERSION=2.12.10 bin/test-multi-jvm-2.1x
-      env: TRAVIS_JDK=11
-      name: "Run multi-jvm tests for Scala 2.12 and Java 11"
+      env: TRAVIS_JDK=8
+      name: "Run multi-jvm tests for Scala 2.12 and Java 8"
     - script: SCALA_VERSION=2.13.0 bin/test-multi-jvm-2.1x
       name: "Run multi-jvm tests for Scala 2.13"
     - script: bin/test-documentation
       name: "Documentation validations and tests"
-    - script: bin/test-docs-code-only
-      env: TRAVIS_JDK=11
-      name: "Documentation tests (Java 11)"
+    ## Until the release is run in JDK11 we must have 100% support for docs
+    ## generation in JDK8 (this means `unidoc` must be run in every PR)
+    - script: bin/test-documentation
+    #- script: bin/test-docs-code-only
+      env: TRAVIS_JDK=8
+      name: "Documentation tests (Java 8)"
```

Comment:
We need an extra job to maintain the JDK8 validation and execution of docs since the release still runs on JDK8. We can only drop this when the release runs on JDK11.